### PR TITLE
Use Secrets Manager at sf-emails-to-s3-exporter

### PR DESCRIPTION
**Pull request by shtukas**
This change is part of the ongoing effort to prevent credentials leaks from support-service-lambdas. Here we are updating holiday-stop-api. 

For a full context of that change see the first PR description: https://github.com/guardian/support-service-lambdas/pull/1949

### CODE REVIEW

File: handlers/sf-emails-to-s3-exporter/src/main/scala/com/gu/sf_emails_to_s3_exporter/Config.scala [hunk 1]
```diff
@@ -19,25 +19,24 @@ case class S3Config(
     bucketName: String,
 )
 
+// case class SalesforceConnectedAppSecrets(authUrl: String, clientId: String, clientSecret: String)
+// case class SalesforceUserSecrets(sfPassword: String, sfToken: String, sfUsername: String)
+
```
Comment (gnm-notona) [inline handlers/sf-emails-to-s3-exporter/src/main/scala/com/gu/sf_emails_to_s3_exporter/Config.scala:24]: can we remove this now?

**Review by gnm-notona — APPROVED** [review]
Just a comment about a comment

File: handlers/sf-emails-to-s3-exporter/src/main/scala/com/gu/sf_emails_to_s3_exporter/Config.scala (hunk 1 above)
Comment (shtukas) [reply handlers/sf-emails-to-s3-exporter/src/main/scala/com/gu/sf_emails_to_s3_exporter/Config.scala:24]: Yes, yes, sorry!

Comment (shtukas) [reply handlers/sf-emails-to-s3-exporter/src/main/scala/com/gu/sf_emails_to_s3_exporter/Config.scala]: Done!

**Comment by shtukas** [comment]
Checked on PROD (not deployable to CODE at the moment)

Screenshot 2023-06-07 at 16 23 40

Screenshot 2023-06-07 at 16 24 08

**Pull request merged**